dissect/target/helpers/regutil.py

```diff
@@ -276,6 +282,9 @@ def add_subkey(self, name: str, key: str):
         """Add a subkey to this key."""
         self._subkeys[name.lower()] = key
 
+    def add_classname(self, value: str):
+        self._class_name = value
+
```

Comment:
```suggestion
```
Perhaps better to do this as just a constructor argument to `VirtualKey`. See the other suggestion.